style.css

```diff
@@ -0,0 +1,358 @@
+  margin: 2%;
+}
+
+.about_section > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.skill_div > h1 {
+  padding-left: 10%;
+}
+
+.framework > h1 {
+  padding-left: 10%;
+}
+
+.grid-container > div > h1 {
+  text-align: left;
+  padding-left: 3%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 3%;
+  margin-bottom: unset;
+}
+
+.about_section > h4 {
+  padding-left: 5%;
+  margin-top: unset;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.diff_classes > li > h4 {
+  background-color: #ebebff;
+  display: inline;
+  padding: 4%;
+  padding-top: 2%;
+  padding-bottom: 2%;
+  border-radius: 8px;
+  margin: 1%;
+  font-size: 17px;
+}
+
+.grid-container > div > div > h4 {
+  margin-top: unset;
+  text-align: left;
+  padding-left: 3%;
+  padding-top: 5%;
+  margin-bottom: unset;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.grid-container > div > div {
+  display: flex;
+  flex-direction: row;
+}
+
+.grid-container > div > div > ul {
+  margin-bottom: unset;
+  padding-left: 4%;
+  margin-top: unset;
+}
+
+.about_section > p {
+  padding-left: 5%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.grid-container > div > p {
+  text-align: left;
+  padding-left: 3%;
+  padding-top: 4%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.diff_classes {
+  margin-top: unset;
+  margin-left: unset;
+  width: 210px;
+}
+
+.about_section > h3 {
+  width: 40%;
+  padding: 4%;
+  padding-left: unset;
+  padding-right: unset;
+  margin-left: 3%;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  text-align: center;
+  border: 1px solid blue;
+}
+
+.grid-container > div > h3 {
+  width: 40%;
+  padding: 4%;
+  padding-left: unset;
+  padding-right: unset;
+  margin-left: 3%;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  text-align: center;
+  border: 1px solid blue;
+}
+
+.grid-item {
+  background-color: rgba(255, 255, 255, 0.8);
+  text-align: center;
+  padding: 10px;
+  font-size: 30px;
+  border-radius: 15px;
+  border: 1px solid lightgray;
+}
+
+.item1 {
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.item2 {
+  grid-column: 1 / span 2;
+  grid-row: 2;
+}
+
+.item3 {
+  grid-column: 1 / span 2;
+  grid-row: 3;
+}
+
+.item4 {
+  grid-column: 1 / span 2;
+  grid-row: 4;
+}
+
```

Comment:
[OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

Reply:
This is optional comment. Will follow this in next project